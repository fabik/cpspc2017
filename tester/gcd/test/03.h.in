100
350 321 186 441 162 194 228 331 274 245 219 315 415 46 240 221 315 490 35 7 449 74 246 183 441 102 87 287 371 41 352 191 422 117 394 183 304 164 209 257 458 221 139 462 226 70 145 268 303 86 277 23 388 133 4 427 462 401 66 322 50 465 235 140 79 435 360 340 3 437 346 216 465 308 394 111 182 449 300 276 287 446 441 8 247 110 305 39 149 457 159 205 492 255 78 378 403 462 137 8
