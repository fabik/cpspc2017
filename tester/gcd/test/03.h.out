1
493
983
1448
1913
2375
2837
3299
3757
4214
4663
5112
5558
5999
6440
6881
7318
7753
8180
8602
9017
9420
9821
10215
10609
10997
11375
11746
12106
12458
12808
13154
13494
13825
14147
14468
14783
15098
15406
15711
16015
16318
16618
16905
17192
17469
17745
18019
18287
18544
18799
19046
19292
19537
19777
20012
20240
20466
20687
20908
21127
21343
21552
21757
21951
22142
22328
22511
22694
22876
23040
23202
23361
23510
23655
23795
23934
24071
24204
24321
24432
24542
24644
24731
24817
24896
24974
25048
25118
25184
25234
25280
25321
25360
25395
25418
25426
25434
25441
25447
